### ISSUE TITLE
Add option for YouTube in VAMC Get updates box

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#15276]
## Description
There is a top down emphasis on storytelling within the VA. 
Related: VAMCs now are able to have YouTube channels
How might we make it possible for VAMCs to add their YouTube link and direct Veterans to this content? 

## Current experience
An existing "Get updates from <name> system health care" box is displayed at the bottom of VAMC pages and connects Veterans with 
- health care news and announcements
- emergency notifications
- operating status
- Facebook
- Twitter
- Flickr
<details>
<summary> Veteran-facing screenshot </summary>

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdjJtIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--e76ba359ccc6bcde517af8ef0dec9850b5f1dae3/image.png)
</details>

---

These links are added at the system level: Email lists and social media 
<details>
<summary> CMS UX </summary>

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBZ3luIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--2c5ae02efa9ed8800734cf282e209054441c1f76/image.png)

</details>

## Related
- VAMC "Get Updates..." links should indicate they open in a new tab #14809

## Tasks
- [ ] TBD

## Acceptance criteria
- [ ] PAO can add a URL for their system's YouTube channel
- [ ] PAO can add unique Link text for their system's YouTube channel

**Comment by mmiddaugh** [comment]
@xiongjaneg can we add this to our next engineering refinement, please?

**Comment by xiongjaneg** [comment]
@mmiddaugh Got it, thank you!

**Comment by swirtSJW** [comment]
Adding a youtube field to the VACM system content type in the CMS would be a low LOE, as would the corresponding FE issue that would involve adding the new field to the query and the template.

**Comment by xiongjaneg** [comment]
Embedding a YouTube videos is out of scope

**Comment by xiongjaneg** [comment]
@xiongjaneg Create separate tickets for front end and then change management.

**Comment by xiongjaneg** [comment]
Michelle: Are analytics automatically baked in or do we need to call that out in tickets as steps or ACs?
Add to the first of these tickets the analytics: Confirm what analytics are running.

**Comment by mmiddaugh** [comment]
@xiongjaneg we forgot to add AC for updating relevant KB article(s) : I think [How to edit a VAMC System homepage](https://prod.cms.va.gov/help/vamc/how-to-edit-a-vamc-system-homepage) is the main one. 

It looks like it's already a general mention which still works, FYI

> Click to expand the Email lists and social media panel to manage your system's social media and GovDelivery information.

**Comment by jilladams** [comment]
Cannot merge until FE is ready to deploy. Jane to add an AC on the FE ticket to merge the CMS PR when FE is ready.
